### PR TITLE
Started implementing analytics

### PR DESCRIPTION
**Pull request by OvidiuGi**

### CODE REVIEW

**Comment by OvidiuGi** [comment]
Work in progress.

File: src/Controller/Admin/MainPageController.php [hunk 1]
```diff
@@ -29,15 +33,44 @@ public function load(): Response
...
+            $newAccountsWithRole = \array_unique($newAccountsWithRole);
+        }
+
+        return $this->render('admin/main_page/analytics/new_accounts.html.twig', [
+            'newAccountsWithRole' => $newAccountsWithRole
+        ]);
+    }
+
+//    /**
+//     * @Route(path="/analytics/admin",methods={"GET"}, name="analytics_admin_logins")
```
Comment (ovidiuenache) [inline src/Controller/Admin/MainPageController.php:71]: comments?

File: templates/admin/main_page/analytics/admin_logins.html.twig [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            ['Apatosaurus (deceptive lizard)', 22.9],
+            ['Archaeopteryx (ancient wing)', 0.9],
+            ['Argentinosaurus (Argentina lizard)', 36.6],
+            ['Baryonyx (heavy claws)', 9.1],
+            ['Brachiosaurus (arm lizard)', 30.5],
+            ['Ceratosaurus (horned lizard)', 6.1],
+            ['Coelophysis (hollow form)', 2.7],
+            ['Compsognathus (elegant jaw)', 0.9],
+            ['Deinonychus (terrible claw)', 2.7],
+            ['Diplodocus (double beam)', 27.1],
```
Comment (ovidiuenache) [inline templates/admin/main_page/analytics/admin_logins.html.twig:46]: what is this? :)))

Comment (OvidiuGi) [reply templates/admin/main_page/analytics/admin_logins.html.twig:46]: :))) woops

**Pull request merged**